packages/bsp/common/usr/lib/armbian/armbian-firstrun

```diff
@@ -53,10 +53,8 @@ case "$1" in
 	sed '1s/^/IMAGE_UUID=/' /proc/sys/kernel/random/uuid >> /etc/armbian-image-release
 
 	# SSH Keys creation
-	rm -f /etc/ssh/ssh_host*
 	read entropy_before </proc/sys/kernel/random/entropy_avail
-	dpkg-reconfigure openssh-server >/dev/null 2>&1
-	service sshd restart
+	dpkg-reconfigure -fnoninteractive openssh-server >/dev/null 2>&1
```

Comment:
```suggestion
	ssh-keygen -A >/dev/null 2>&1
```

It seem like this will block. Maybe it try to start `ssh.service` which should run after this script. So it fall into a loop: `dpkg-reconfigure` is waiting `ssh.service` to start while `ssh.service` is waiting `armbian-firstrun.service` to finish.

We can use `sshkey-gen` to generate host keys. Accroding to my test, this work fine without timeout